scripts/minikube-helpers.sh

```diff
@@ -101,7 +101,7 @@ function check_prometheus_installation() {
 	kubectl_tool=$(which kubectl)
 	check_err "Error: Please install the kubectl tool"
 	# Check to see if kubectl supports kustomize
-	kubectl kustomize --help >/dev/null 2>/dev/null
+	kubectl --help | grep "kustomize" > /dev/null
```

Comment:
Bash sometimes interprets the space after '>`, so I suggest you change it to the below in both places
```
kubectl --help | grep "kustomize" >/dev/null
```

Reply:
Thanks for the review @dinogun I have made the changes accordingly.